cmd/ipfs/daemon.go

```diff
@@ -12,6 +12,7 @@ import (
 	_ "github.com/ipfs/go-ipfs/Godeps/_workspace/src/github.com/codahale/metrics/runtime"
 	ma "github.com/ipfs/go-ipfs/Godeps/_workspace/src/github.com/jbenet/go-multiaddr"
 	"github.com/ipfs/go-ipfs/Godeps/_workspace/src/github.com/jbenet/go-multiaddr-net"
+	prom "github.com/prometheus/client_golang/prometheus"
```

Comment:
want to run `make vendor` to make sure imports are re-written. if godeps yells at you, see https://github.com/ipfs/go-ipfs/pull/1385#issuecomment-113139449